### ISSUE TITLE
Fix bloom and add skybox

### ISSUE THREAD

**Issue by VERON0990** [Open4Es/Open4Es-Shader-Android#90]
Currently in shaders including the blur effect, the sky turns white, I find it necessary to make a skybox! I also wanted to add opacity on the shadows, that is, changing the value of the shadow becomes darker, and also with the effect of blur, changing the value of transparency should result in even greater blurring but decreasing brightness. Sorry for my english i'm russian.
https://github.com/wwwtyro/glsl-atmosphere

**Comment by VERON0990** [comment]
![Uploading Screenshot_2021-07-11-11-06-14-80_bab8561a5f94f0a53956d8cbbeaad774.jpg…]()

**Comment by ENDERMANYK** [comment]
nah, due to GL4ES's limitation its very hard to draw a properly sky to the screen..
